.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Pull request
+about: Create a pull request to contribute new features or bug fixes
+title: "[PULL REQUEST]"
```

Comment:
What does this tag do?

Reply:
I think this is just the comment section which won't show up in the actual template, similar to our existing ones in the issue templates.